src/example_agents/basic_agent.py

```diff
@@ -0,0 +1,116 @@
+from typing import List, Type
+
+from pydantic import Field
+from steamship.agents.functional import FunctionsBasedAgent
+from steamship.agents.llms.openai import ChatOpenAI
+from steamship.agents.mixins.transports.slack import (
+    SlackTransport,
+    SlackTransportConfig,
+)
+from steamship.agents.mixins.transports.steamship_widget import SteamshipWidgetTransport
+from steamship.agents.mixins.transports.telegram import (
+    TelegramTransport,
+    TelegramTransportConfig,
+)
+from steamship.agents.schema import Tool
+from steamship.agents.service.agent_service import AgentService
+from steamship.agents.tools.image_generation.stable_diffusion import StableDiffusionTool
+from steamship.invocable import Config
+from steamship.utils.repl import AgentREPL
+
+
+class BasicAgent(AgentService):
```

Comment:
It looks like example_agents has become in practice example_agentservices / example_packages; I worry about the name collision here because we also support custom Agents.

Reply:
Just went through and renamed all of the examples from SomethingAgent to SomethingAgentService